make-uniform/production/Build Standard Database.Rmd

```diff
@@ -10,53 +10,77 @@ output:
 ## Administration
 
 #### Purpose
-Procedure to translate any number of on-board survey data sets into a single dataset with common variables and common responses.  This is a work-in-progress production version of the procedures.  See ``../mini-example`` and ``../small-example`` for working proto-types.  In this script, we put in place procedures to process surveys into a standard database.  See ``Extract Variables from Legacy Surveys.Rmd`` for procedures to extract variables from legacy surveys (i.e., those with SAS summary scripts). 
+Procedure to translate any number of on-board survey data sets into a single 
+dataset with common variables and common responses.  This is a work-in-progress 
+production version of the procedures.  See ``../mini-example`` and 
+``../small-example`` for working proto-types.  In this script, we put in place 
+procedures to process surveys into a standard database.  See 
+``Extract Variables from Legacy Surveys.Rmd`` for procedures to extract 
+variables from legacy surveys (i.e., those with SAS summary scripts). 
 
 #### _ISSUES_
 1.  Work carefully through first few datasets, as the code is a bit coarse in places
 
 #### _TODO_
+1. New users should add their username and relative file path to 
```

Comment:
@jhelsel11: Given that this database will be rather small, may it be easier just to list the user names and dirs here and build the dataframe rather than using a separate CSV?